### PR TITLE
e2e conformance test against local cluster

### PR DESCRIPTION
**Pull request by dims**

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -9918,6 +9918,22 @@
       "sig-scalability"
     ]
   },
+  "ci-kubernetes-local-e2e": {
+    "args": [
+      "--up",
```
Comment (krzyzacy) [inline jobs/config.json]: --up and --test are default true

File: jobs/config.json [hunk 2]
```diff
@@ -9918,6 +9918,22 @@
...
+      "--provider=local",
+      "--deployment=local",
+      "--test",
+      "--extract=ci/latest",
+      "--ginkgo-parallel=1",
+      "--test_args=--ginkgo.focus=\\[Feature:Conformance\\]",
+      "--timeout=320m"
```
Comment (krzyzacy) [inline jobs/config.json]: I don't think your conformance test should take 320m?

**Comment by BenTheElder** [comment]
please use `hack/update-config.sh` to patch the security repo jobs (this is a somewhat recent change tested by `pull-test-infra-verify-config`)

**Comment by BenTheElder** [comment]
oh actually that's strange, *CI* should not trigger this, i thought you were still doing a presubmit when I saw the failure. this is probably a config_sort failure then (also part of that job)

**Comment by krzyzacy** [comment]
I bet you just need to run config sort `bazel run //jobs/config_sort.py`

**Comment by BenTheElder** [comment]
> I bet you just need to run config sort bazel run //jobs/config_sort.py

FWIW update-config will do this too. shrug.
Any more nits otherwise sen? this seems fine to me

**Comment by krzyzacy** [comment]
yeah, we can try it out.

**Comment by dims** [comment]
thanks, ran the `hack/update-config.sh` script and updated the commit

File: jobs/config.json (hunk 1 above)
Comment (dims) [reply jobs/config.json]: fixed

File: jobs/config.json (hunk 2 above)
Comment (dims) [reply jobs/config.json]: dropped it down to 120 :)

**Comment by BenTheElder** [comment]
> AssertionError: jobs: ('insufficient kubetest leeway', u'ci-kubernetes-local-e2e', 120, 120), prow timeout need to be at least 20min longer than timeout in config.json

 ¯\\\_(ツ)\_/¯

**Comment by dims** [comment]
all green now

**Pull request merged**